spring-cloud-task-starter/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-task-parent</artifactId>
+		<version>1.0.1.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>spring-cloud-task-starter</artifactId>
+	<packaging>jar</packaging>
+	<name>Spring Cloud Task Starter</name>
+	<description>Spring Boot starter for Spring Cloud Task</description>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-task-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
This is based on a conversation with @philwebb.  Both those dependencies have everything else optional so they actually should be very light additions.  This approach also allows you to add this starter with the appropriate starter (say batch or stream starters) and the autoconfig would kick in.